.github/workflows/ghstack_land.yaml

```diff
@@ -0,0 +1,48 @@
+name: Land ghstack PRs
+on:
+  repository_dispatch:
+    types: [land-command]
+
+jobs:
+  ghstack_land:
+    name: ghstack Land
+    runs-on: ubuntu-latest
+    steps:
+    - name: Show Environment Variables
+      run: env
+    - name: Show Github Object
+      run: |
+        cat <<'EOF'
+        ${{ toJson(github) }}
+        EOF
+    - name: Show Github Event Path Json
+      run: 'cat $GITHUB_EVENT_PATH || true'
+    - uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.LAND_TOKEN }}
+        fetch-depth: '0'
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+    - name: Check Current CI Status
+      run: |
+        pip install requests ghstack
+        .github/workflows/scripts/ghstack-perm-check.py <<'EOF'
+        ${{ toJson(github) }}
+        EOF
+      env:
+        GITHUB_TOKEN: ${{ secrets.LAND_TOKEN }}
+    - name: Land It!
+      run: |
+        git config --global user.email "foundation-model-stack@googlegroups.com"
```

Comment:
we might want to use an email that we use for pypi for this

Reply:
Yeah, that makes sense. I can add an extra email address to the account and configure it here if you share the email address with me.

Do you have any existing mechanism to share secrets within the maintainer team? I would like to share the credentials of the bot account so that everyone may join if needed.

For now I set them up in 1password, including 2FA.